appengine/standard_python3/building-an-app/building-an-app-3/templates/index.html

```diff
@@ -35,6 +35,7 @@
       alert(msg);
     }
   </script>
+  <script src="https://www.gstatic.com/firebasejs/8.9.1/firebase-auth.js"></script>
```

Comment:
Should this line be inside a region tag?

Reply:
No, the region tag highlights another dependency.